assets/src/edit-story/app/media/utils/useTranscodeVideo.js

```diff
@@ -124,9 +124,12 @@ function useTranscodeVideo() {
     currentUser.meta?.web_stories_media_optimization
   );
 
+  const ffmpegSupported = Boolean(window?.crossOriginIsolated);
```

Comment:
Let's just move this to `isFeatureEnabled` please.

Can rename it to `isFeatureSupported` if we want, but I don't want to introduce a separate variable for it.

That also means no change is needed to `uuseUploadMedia`

Reply:
Fixed